source/manual/rummager-traffic-replay.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#search-team"
+title: 'Replay HTTP traffic on rummager machines'
+parent: "/manual.html"
+layout: manual_layout
+section: Backups
+last_reviewed_on: 2017-09-05
+review_in: 3 months
+---
+
+Have have started using [GOR](https://github.com/buger/goreplay) to store POST and DELETED requests
+to the rummager servers.
+
+At is stored on the individual machine at:
+
+```
+/var/log/gor_dumps
+```
+
+If the index should fail and we need to restore from backup we can then replay this traffic rather than
```

Comment:
Since this is the only use case we're considering, maybe we should title the page something like "Replay search indexing traffic after restoring a backup".

Then we can move this right to the top of the page.

We should also link to this page from https://docs.publishing.service.gov.uk/manual/elasticsearch-dumps.html

Reply:
done